### PR TITLE
Fix OSTree check

### PR DESCRIPTION
**Pull request by kjbracey-arm**
OSTree script is failing due to the presence of /run/ostree-booted
in initramfs.

Issue raised at

https://github.com/ostreedev/ostree/issues/2485

Delete /run/ostree-booted from initramfs to make it work.

Also remove some stray debugging.

### CODE REVIEW

File: meta-lmp-support/recipes-core/initrdscripts/initramfs-framework/ostreecheck
Comment (JanneKiiskila) [inline meta-lmp-support/recipes-core/initrdscripts/initramfs-framework/ostreecheck:23]: Should there be some kind of "backwards compatibilty / future proofing"? I.e. check if the fiel exists and only then remove it? Would it fail it the file didn't exist?

Comment (kjbracey-arm) [reply meta-lmp-support/recipes-core/initrdscripts/initramfs-framework/ostreecheck:23]: This would print an error if it didn't exist, but continue.

Probably safer to move it out of the way then put it back (and send errors to `/dev/null`). I'll do that. Our initramfs `/run` actually becomes the real `/run` later, it's not chucked. But then something later in boot currently recreates `/rev/ostree-booted` manually anyway, so my deletion still works for now.

**Comment by kjbracey-arm** [comment]
An alternative would be to apply this pending patch to ostree: https://github.com/ostreedev/ostree/pull/2486. Going to test that, but should probably wait until it's merged.

**Review by JanneKiiskila — APPROVED** [review]

**Review by kivaisan — APPROVED** [review]
Tested and looks ok.

**Pull request merged**
